### PR TITLE
John/analytics fixes

### PR DESCRIPTION
**Pull request by jordajm**
### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything

### Description:

- Updates event names based on Matt's feedback in the spreadsheet
- Adds events for the transaction flow that I missed before

### CODE REVIEW

File: translations/all-messages.json [hunk 1]
```diff
@@ -197,6 +197,8 @@
   "message.enable": "Enable Messaging",
```
[the remark this answers is not in the document]
Comment (jordajm) [reply translations/all-messages.json]: hmmm - that must've happened when I merge master into this branch. Should be fine - maybe Micah wasn't running the dapp locally when he made his last commit or something...

[the remark this answers is not in the document]
Comment (micahalcorn) [reply translations/all-messages.json]: Indeed, the `messaging-provider` string ids should not be added back.

[the remark this answers is not in the document]
Comment (franckc) [reply translations/all-messages.json]: OOOPS. @jordajm I did merge before @micahalcorn commented. so those strings got added back...

[the remark this answers is not in the document]
Comment (micahalcorn) [reply translations/all-messages.json]: It's not a big deal. @jordajm do you know of a command that can be run or a way to purge unused strings in the all-messages?

[the remark this answers is not in the document]
Comment (jordajm) [reply translations/all-messages.json]: @micahalcorn - `all-messages.json` gets re-created from scratch every time you change a string ID or default message. Here's the webpack plugin I made to do that:
https://github.com/OriginProtocol/origin-dapp/blob/master/translations/scripts/prepareMessagesPlugin.js#L33

**Pull request merged**